README.md

```diff
@@ -1,7 +1,20 @@
 # JigglypuffRL
 
+## Installation
 ```
 conda env update --name JigglypuffRL --file environment.yml
 conda activate JigglypuffRL
-python -m JigglypuffRL.deeprl.ppo1.ppo1
+
+pip install jigglypuff-rl
```

Comment:
Should we change the temp env name we create here to avoid any confusions with the package?
Maybe something like 
```
conda env update --name JgRL --file environment.yml
conda activate JgRL

pip install jigglypuff-rl
```

What do you think?